Source/Calling/CallParticipantSnapshot.swift

```diff
@@ -21,6 +21,13 @@ import Foundation
 class CallParticipantsSnapshot {
     
     public private(set) var members : OrderedSetState<AVSCallMember>
+
+    // We take the worst quality of all the legs
+    public var networkQuality: NetworkQuality {
+        return members.map { $0.networkQuality }
```

Comment:
You can use they keypath syntax here:

```suggestion
        return members.map(\.networkQuality)
```